test/gtest/ucp/test_ucp_wireup.cc

```diff
@@ -632,7 +632,7 @@ UCS_TEST_P(test_ucp_wireup_1sided, disconnect_nonexistent) {
     sender().destroy_worker();
 }
 
-UCS_TEST_P(test_ucp_wireup_1sided, disconnect_reconnect) {
+UCS_TEST_P(test_ucp_wireup_1sided, disconnect_reconnect, "RNDV_THRESH=inf") {
```

Comment:
1. do we need this for all tests in this test suite?
2. pls add comment why it's needed

Reply:
> 1. do we need this for all tests in this test suite?

Rest cases send small messages, e.g. 0 or 8 bytes. Do you propose to set "RNDV_THRESH=inf" in `test_ucp_wireup_1sided::init()`?
> 2. pls add comment why it's needed

ok

Reply:
> Rest cases send small messages, e.g. 0 or 8 bytes. Do you propose to set "RNDV_THRESH=inf" in `test_ucp_wireup_1sided::init()`?

yes

Reply:
[here](https://github.com/openucx/ucx/blob/eed3d5668fed4020804124eb0fcb6a5e93f83534/test/gtest/ucp/test_ucp_wireup.cc#L559) is an exception which may lead to confusion, IMO we can create a new class to override `init()` there or continue `UCS_TEST_P` using to keep these tests aligned